lib/migration.js

```diff
@@ -770,6 +770,28 @@ function mixinMigration(MySQL, mysql) {
     return dt;
   };
 
+  function columnDefault(p, line) {
+    if (typeof p.type === 'undefined' || typeof p.type.name !== 'string') {
```

Comment:
Is this additional check required?
```
typeof p.type.name !== 'string
```

IMO, it will be causing issues with Number fields.